stockAndPrices/services/app/main.py

```diff
@@ -14,10 +19,13 @@ def login():
     if request.method == "GET":
         return "<h1>Login Page</h1>"
     else:
-        user = request.form['user_name']
+        user = request.form.get('username')
+        #pwd = request.form.get('password')
+        #print(user)
```

Comment:
Let us remove the commented out print statements, the password field we can leave as we will use it later

Reply:
Removed